target/linux/ath79/dts/ar9344_atheros_db120.dts

```diff
@@ -0,0 +1,205 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+
+#include "ar9344.dtsi"
+
+#include <dt-bindings/gpio/gpio.h>
+#include <dt-bindings/input/input.h>
+
+/ {
+	model = "Atheros DB120 reference board";
+	compatible = "atheros,db120", "qca,ar9344";
+
+	aliases {
+		led-boot = &led_system;
+		led-failsafe = &led_system;
+		led-running = &led_system;
+		led-upgrade = &led_system;
+	};
+
+	leds {
+		compatible = "gpio-leds";
+
+		wlan2g {
+			label = "green:wlan2g";
+			gpios = <&gpio 13 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy0tpt";
+		};
+
+		wlan5g {
+			label = "green:wlan5g";
+			gpios = <&gpio 12 GPIO_ACTIVE_LOW>;
+			linux,default-trigger = "phy1tpt";
+		};
+
+		led_system: system {
+			label = "green:system";
+			gpios = <&gpio 14 GPIO_ACTIVE_LOW>;
+			default-state = "on";
```

Comment:
This line can be dropped since you have the `led-running` alias pointing to `&led_system`, thus it should be on by default already. Curiously, this is still declared for many ath79 devices, though technically it is probably no longer needed.

Reply:
No. default-state will be evaluated very early in the boot process, even before the user-space diag.sh script (using the led-* definitions) is used at all. Apart from that, led-running is only evaluated _after_ the boot process is finished.

It will light the LED even before led-boot is doing anything and thus will give you another hint about in which step of the boot process you are.

This should be kept.

Reply:
So this is for bootloaders not initialising the LED before starting the kernel - good to know, will have to check my current PR, thanks!

Reply:
Well, AFAIK you have three choices:
"on" put it on no matter what the bootloader does
"off" put if off no matter what the bootloader does (the default)
"keep" keep the state it's in from the bootloader

Reply:
@s-2 Something for the wiki